src/css/custom.css

```diff
@@ -508,6 +508,13 @@ html[data-theme='dark'] .DocSearch-Hit a {
   }
 }
 
+.github-codeblock-reference-link {
+  margin-top: -16px;
```

Comment:
Negatives are generally not recommended, but I guess we don't have the ability to reduce the margin/padding on the actual code block 🤔

Reply:
We could theoretically add an additional class to the wrapping div on both and use that as a selector to reduce the margin/padding on the code block, but I think this would potentially be overkill to avoid this simpler approach.